src/Slider/components/Bar/index.js

```diff
@@ -0,0 +1,71 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+
+import { left as leftRight } from "../../../utils/rtl";
+import defaultTheme from "../../../defaultTheme";
+import type { Ref } from "../../../common/common.js.flow";
+
+import type { CalculateBarPosition, Props } from "./index";
+
+const StyledBar = styled.div`
+  display: block;
+  position: relative;
+  width: 100%;
+  height: 24px;
+  cursor: pointer;
+  -webkit-tap-highlight-color: transparent;
+  -webkit-touch-callout: none;
+  -webkit-user-select: none;
+`;
+
+export const calculateBarPosition: CalculateBarPosition = (value, max, min, hasHistogram) => {
+  if (Array.isArray(value)) {
+    return {
+      left: +(((value[0] - min) / (max - min + 1)) * 100).toFixed(1),
+      width: +(((value[value.length - 1] - value[0] + 1) / (max - min + 1)) * 100).toFixed(1),
+    };
+  }
+  const addition = hasHistogram ? 1 : 0;
+  return {
+    left: 0,
+    width: +(((value - min + addition) / (max - min + addition)) * 100).toFixed(1),
+  };
+};
+
+export const StyledBarPart = styled(({ width, left, theme, active, ...props }) => (
+  <div {...props} />
+)).attrs(({ width, left, theme }) => {
+  return {
+    style: {
+      width: `${width}%`,
+      [leftRight({ theme })]: `${left}%`,
+    },
+  };
+})`
+  position: absolute;
+  height: 4px;
+  top: 10px;
+  border-radius: 4px;
+  background-color: ${({ theme, active }) =>
+    active ? theme.orbit.paletteProductNormal : theme.orbit.paletteInkLighter};
+  z-index: ${({ active }) => (active ? 30 : 20)};
```

Comment:
I thought we started using new `z-index` values from tokens, is that just a draft still?

Reply:
It's still draft and we don't have tokens for it, yet. But the values are actually are the proposed levels.